src/pages/PageHome.vue

```diff
@@ -1,13 +1,112 @@
 <template>
-  <q-page class="flex flex-center">
-    <h5>Home Page</h5>
+  <q-page class="constrain q-pa-md">
+    <div class="row q-col-gutter-lg">
+      <div class="col-12 col-sm-8">
+        <q-card
+          v-for="post in posts"
+          :key="post.id"
+          class="card-post q-mb-md"
+          flat
+          bordered
+        >
+          <q-item>
+            <q-item-section avatar>
+              <q-avatar>
+                <img src="https://cdn.quasar.dev/img/boy-avatar.png" />
+              </q-avatar>
+            </q-item-section>
+
+            <q-item-section>
+              <q-item-label class="text-bold">Oleg_Nesterov</q-item-label>
+              <q-item-label caption>
+                {{ post.location }}
+              </q-item-label>
+            </q-item-section>
+          </q-item>
+
+          <q-separator />
+          <q-img :src="post.imageUrl" />
+          <q-card-section>
+            <div>{{ post.caption }}</div>
+            <div class="text-caption text-grey">{{ getDate }}</div>
+          </q-card-section>
+        </q-card>
+      </div>
+      <div class="col-4 large-screen-only">
+        <q-item class="fixed">
+          <q-item-section avatar>
+            <q-avatar size="50px">
+              <img src="https://cdn.quasar.dev/img/boy-avatar.png" />
+            </q-avatar>
+          </q-item-section>
+
+          <q-item-section>
+            <q-item-label class="text-bold">Oleg_Nesterov</q-item-label>
+            <q-item-label caption> Oleg_Nesterov </q-item-label>
+          </q-item-section>
+        </q-item>
+      </div>
+    </div>
   </q-page>
 </template>
 
 <script>
 import { defineComponent } from 'vue'
+import { date } from 'quasar'
+import { filter } from 'compression'
 
 export default defineComponent({
-  name: 'PageHome'
+  name: 'PageHome',
+  data () {
+    return {
+      posts: [
```

Comment:
I do not like the way you are making this mock object in your component, make minor refactoring: decomposition is a good choice for this case